package.json

```diff
@@ -28,8 +28,8 @@
     "publish-next": "lerna publish --force-publish=\"*\" --pre-dist-tag next --preid next",
     "test-api": "lerna run test-api",
     "test-coverage": "lerna run test-coverage",
-    "test-types": "lerna run test-types",
-    "test": "yarn build && yarn lint && yarn test-coverage"
+    "test-types": "lerna --concurrency 1 run test-types",
```

Comment:
[Concurrent dtslint invocations](https://github.com/lerna/lerna/blob/main/core/global-options/README.md#--concurrency) were trampling each other installing TypeScript versions ... At least I *think* that's what was happening [here](https://github.com/mdx-js/mdx/pull/1622/checks?check_run_id=3451812571#step:5:373) -- and [here](https://github.com/mdx-js/mdx/issues/1172) ... Perhaps [here](https://github.com/microsoft/dtslint/issues/267) too ...